modules/data-streams/src/test/java/org/elasticsearch/datastreams/DataStreamsStatsTests.java

```diff
@@ -269,7 +268,9 @@ private DataStreamsStatsAction.Response getDataStreamsStats(boolean includeHidde
         DataStreamsStatsAction.Request request = new DataStreamsStatsAction.Request();
         if (includeHidden) {
             request.indicesOptions(
-                new IndicesOptions(EnumSet.of(IndicesOptions.Option.ALLOW_NO_INDICES), EnumSet.of(IndicesOptions.WildcardStates.HIDDEN))
+                IndicesOptions.builder(request.indicesOptions())
+                    .wildcardOptions(IndicesOptions.WildcardOptions.builder(request.indicesOptions().wildcardOptions()).includeHidden(true))
+                    .build()
```

Comment:
The previous line seems to be simply setting the options directly, is it more correct for us to modify the existing options or should we be discarding them here?

Reply:
It think that's a fair enough explanation, yeah